deploy/ocp-templates/glusters3template.json

```diff
@@ -0,0 +1,205 @@
+{
+    "kind": "Template",
```

Comment:
Also move this file into the examples directory. It should not be in `deploy/ocp-templates` unless/until it is used by a script.